airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -622,6 +624,7 @@ def __init__(
         self.configuration_overrides = configuration_overrides
         self.wait_for_completion = wait_for_completion
         self.config = config or {}
+        self.name = name or self.config.pop("name", f"emr_serverless_job_airflow_{str(uuid4())}")
```

Comment:
Small nit I just realized, you shouldn't need the `str(...)` around the uuid, since the f-string will cast it to a string for you.